app/assets/stylesheets/application.css.scss

```diff
@@ -15,46 +15,50 @@
  *= require font-awesome
  */
 
- .social-links a {
-   text-align: center;
-   display: inline-block;
-   width: 36px;
-   height: 36px;
-   border: 1px solid white;
-   border-radius: 100%;
-   margin-right: 7px; /*space between*/
- }
-
- .social-links a i {
-   font-size: 20px;
-     line-height: 38px;
-   color: white;
- }
-
- #facebook {
-     background: #3B5998;
- }
-
- #twitter {
-     background: #6CDFEA;
- }
-
- #instagram {
-     background: #e74c3c;
- }
+* {
+  font-family: "Open Sans";
+}
+
+.social-links a {
+  text-align: center;
+  display: inline-block;
+  width: 36px;
+  height: 36px;
+  border: 1px solid white;
+  border-radius: 100%;
+  margin-right: 7px; /*space between*/
+}
+
+.social-links a i {
```

Comment:
Merge rule `.social-links a i` with rule on line 22<br>Selector should have depth of applicability no greater than 2, but was 3